elizabeth/homework1.js

```diff
@@ -0,0 +1,173 @@
+/* Section 1
+a) True -- == is not exact and JS will auto convert "1" to a number.
+
+b) False -- === is exact, the values must match in type and value.
+
+c) True -- for the same reason "a" was true, both value and type are not checked, however it does not actually return a value.
+
+d) False -- the right side would be "undefined" and x!== undefined
+
+e) False -- empty string is "undefined" and the right side is a string equal to a space.
+
+f) True -- if x has a value of a string or a number, it is truthy, False if x has a value of Nan, empty string, or undefined.
+
```

Comment:
In this case, true is returned because x has been literally set to true, regardless of its previous state. It's like everything before it was wiped clean.